services/redis-cluster/README.md

```diff
@@ -0,0 +1,5 @@
+# If you need REDIS
+
+The top level `./manage` script does not stand up the redis cluster by default, it must be started manually. 
+
```

Comment:
Is this the issue I’m having?  If so, I don’t understand what it is telling me to do. Perhaps be more explicit at telling me what to do vs. pointing to some code?